combat-trainer.lic

```diff
@@ -2613,6 +2613,10 @@ class AttackProcess
 
       bput(command, 'Wouldn\'t it be better if you used a melee weapon?', 'Face What?', 'Roundtime', "You aren't close enough to attack", 'It would help if you were closer', 'There is nothing else to face!', 'You must be hidden', 'flying too high for you to attack', 'Bumbling, you slip')
     end
+    
+    if reget(5, 'Wouldn\'t it be better if you used a melee weapon?')
```

Comment:
Not a big deal, but I think doing a `case when` on bput makes more sense here instead of reget

Reply:
Yes, made sense to me but the rest of this section also used reget. So I stuck with the current implementation.

Reply:
Yup, which is what I did at first, it's weird, because those have multiple lines which come out and sometimes fail to get the proper text. The command you're using should pull it well enough, but definitely if you make the change watch it carefully, it could fail. But if we simply don't do brawling, then we don't need to worry about this at all.